app/Http/Controllers/ChopsController.php

```diff
@@ -138,30 +141,24 @@ public function show($id)
 	 */
 	public function edit($id)
 	{
-		//
-	}
+        $owner_id = Chop::find($id)->user_id;
+        $user = Auth::user();
 
-	/**
-	 * Update the specified resource in storage.
-	 *
-	 * @param  Request  $request
-	 * @param  int  $id
-	 * @return Response
-	 */
-	public function update(Request $request, $id)
-	{
-		//
+        if($user->id === $owner_id) {
+            $chops = Chop::find($id);
+
+            return view('chops.updatechops', compact('chops', 'id'));
+        }
+        else
+            return redirect('chops');
```

Comment:
Follow a convention and stick by it. Let the `else` have open and closing curly braces